Doc/library/venv.rst

```diff
@@ -143,7 +143,7 @@ creation according to their needs, the :class:`EnvBuilder` class.
 
 .. class:: EnvBuilder(system_site_packages=False, clear=False, \
                       symlinks=False, upgrade=False, with_pip=False, \
-                      prompt=None, upgrade_deps=False)
+                      prompt=None, upgrade_deps=False, \*, gitignore=False)
```

Comment:
As Edgar mentioned we shouldn't need the backslash escape here (the ``*`` is parsed by Sphinx distinctly from reStructuredText markup):
```suggestion
                      prompt=None, upgrade_deps=False, *, gitignore=False)
```